Src/Notion.Client/Models/PropertyValue/DatePropertyValue.cs

```diff
@@ -14,7 +14,7 @@ public class DatePropertyValue : PropertyValue
         /// Date
         /// </summary>
         [JsonProperty("date")]
-        public Date Date { get; set; }
+        public Date? Date { get; set; }
```

Comment:
@kashifsoofi you can remove the nullable as it is a class. rest LGTM.

Reply:
Good spot, done